main.py

```diff
@@ -36,15 +36,19 @@ def main():
         if not job_data:
             logging.info('No pending job requests.')
             return
-        classifier_data = job_services.train_classifier(
-            job_data['algorithm_id'], job_data['training_data'])
-        status = job_services.store_job_result(
-            job_data['job_id'], classifier_data)
+        frozen_model_proto = job_services.train_classifier(
+            job_data['algorithm_id'], job_data['algorithm_version'],
+            job_data['training_data'])
 
-        if status != 200:
-            logging.warning(
-                'Failed to store result of the job with \'%s\' job_id',
-                job_data['job_id'])
+        if frozen_model_proto:
```

Comment:
Under what circumstances would this not hold?

Reply:
It will hold for all the cases now. Earlier the `algorithm_version` check happened when training the classifier (and it would hold if `algoritm_version` had a mismatch) but now I moved it to job validation checks. Hence, this condition is removed now as it will always hold.